lib/command.html

```diff
@@ -1,4 +1,10 @@
-<div class="ZenCommand">
+<div
```

Comment:
I don't think it makes sense to be able to move the command palette. Because it is only on the screen temporarily it doesn't block the ui in the same way as the bottom test lister. And adding the crab cursor makes it confusing with other interactions like focusing the text box and selecting options. I think we should not make this moveable.